api/src/connection/datasource.ts

```
@@ -1,23 +1,24 @@
-import { DataSource, DataSourceOptions, DefaultNamingStrategy } from 'typeorm';
+import { DataSource, DataSourceOptions } from 'typeorm';
 import * as process from 'process';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)